### PR TITLE
VIH-9512 - background service

### PR DESCRIPTION
**Pull request by marcogagliardi**
https://tools.hmcts.net/jira/browse/VIH-9512

added background service for remove heartbeats and RemoveInstantMessagesForConference

### CODE REVIEW

File: VideoApi/VideoApi/Controllers/ConferenceController.cs [hunk 1]
```diff
@@ -26,6 +27,7 @@
 using VideoApi.Services.Exceptions;
 using VideoApi.Services.Mappers;
 using VideoApi.Validations;
+using ICommand = System.Windows.Input.ICommand;
```
Comment (oliver-scott) [inline VideoApi/VideoApi/Controllers/ConferenceController.cs:30]: not needed

File: VideoApi/VideoApi/Startup.cs [hunk 2]
```diff
@@ -8,6 +8,7 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Authorization;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
```
Comment (oliver-scott) [inline VideoApi/VideoApi/Startup.cs:11]: not needed

**Review by oliver-scott — COMMENTED** [review]
A couple of unneeded using statements and a typo in one of the filenames (Background**Porcess**Queue), otherwise looks good

File: VideoApi/VideoApi/Controllers/ConferenceController.cs (hunk 1 above)
Comment (marcogagliardi) [reply VideoApi/VideoApi/Controllers/ConferenceController.cs:30]: removed

File: VideoApi/VideoApi/Startup.cs (hunk 2 above)
Comment (marcogagliardi) [reply VideoApi/VideoApi/Startup.cs:11]: removed

**Review by oliver-scott — APPROVED** [review]

**Pull request merged**